tools/docker-builder/types.go

```diff
@@ -93,6 +93,15 @@ func DefaultArgs() Args {
 		"istioctl",
 		"operator",
 		"install-cni",
+		"base",
```

Comment:
The rest of the change is great - thanks for the fix - but I don't think we want to build all of these every time. Currently we are only rebuilding these when there are CVEs in the base image since they are slow + make a bunch of external calls. There is also a `distroless` one fwiw

Reply:
OK, I think it can be removed from the list for what you said.

Reply:
Thanks.